xen/arch/arm/tee/ffa_private.h

```diff
@@ -240,6 +240,36 @@
 #define FFA_NOTIFICATION_INFO_GET_32    0x84000083U
 #define FFA_NOTIFICATION_INFO_GET_64    0xC4000083U
 
+/**
+ * Encoding of features supported or not by the fw in a 64bit value:
+ * - Function IDs are going from 0x60 to 0x86 so we have a maximum of 38 so
```

Comment:
The reserved range is from 0x60 to 0xff, but 64 bits should hopefully be enough.

Reply:
Yes i agree, i wanted to prevent to use a bitmap as the maximum number right now is low enough.